### DISCUSSION TITLE
No response from sales

### DISCUSSION THREAD

**Discussion by yushixu-emodo** [community/community#68761] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Similar to #62394. I contacted sales twice by the online form in order to add more licenses for our enterprise subscription, but I have not received any responses after a week, other than the "Hi from GitHub Enterprise". Is this normal? I am really not a fan of needing such complex communication just to add a few licenses.

**Comment by jonas-ekstroem** [comment]
Same here. I've sent in a request several times with no response. What would a potential customer do to get some attention from the busy sales team?

**Comment by netcivil** [comment]
did anyone find a way to get in touch with sales?

**Comment by yushixu-emodo** [comment]
I finally got contacted by an account manager via another person who once joined one of the sales calls (can't remember if he is sales or also an account manager.) I would call this "luck" and don't think this is a reproducible approach
